assets/src/edit-story/components/panels/sizePosition.js

```diff
@@ -47,6 +47,14 @@ const BoxedNumeric = styled(Numeric)`
   border-radius: 4px;
 `;
 
+const StyledLocked = styled(Locked)`
+  margin: 0 10px;
+`;
+
+const StyledUnlocked = styled(Unlocked)`
+  margin: 0 10px;
+`;
```

Comment:
Wouldn't a single wrapper be DRY'er here?

```js
const Wrapper = styled.span`
  margin: 0 10px;
`;
```

And then further down:
```js
icon={<Wrapper><Locked /></Wrapper>}
uncheckedIcon ={<Wrapper><Unlocked /></Wrapper>}
```

It's of course an unnecessary DOM element, which I'm also not in favor of.

Alternatively it could be a `styled.css` declaration:

```js
const withMargin = css`margin: 0 10px;`;
const StyledLocked = styled(Locked)`${withMargin}`;
const StyledUnlocked = styled(Unocked)`${withMargin}`;
```